CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 
 ## [Unreleased]
+### Changed
+- Commands - Drop environment variables (breaking change!).
```

Comment:
[Code Convention] This will be merge in v3 so you can remove the breaking change (As we did not yet put in place the new changelogs ;) )

Reply:
:+1: